src/vuex/modules/connection.js

```diff
@@ -18,6 +18,16 @@ export default function({ apollo }) {
   }
 
   const actions = {
+    async getNetworkByAddress(store, address) {
+      const { data } = await apollo.query({
+        query: Networks,
+        fetchPolicy: "cache-first"
+      })
+      const network = data.networks.find(
+        network => address.indexOf(network.address_prefix) == 0
+      )
+      return network || false
```

Comment:
you are returning 2 different types here. Instead of returning false you should return undefined and check in the receiving function to make the API predictable